test/models/value_set_test.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class ValueSetTest < ActiveSupport::TestCase
```

Comment:
This should probably have tests in it, or be deleted if it's not going to contain anything, right?

Reply:
resolved